webapi-pip/index.html

```diff
@@ -0,0 +1,119 @@
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-abr.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-drm.js"></script>
+    <script type="text/javascript"
+        src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-container-mp4.js"></script>
+    <script type="text/javascript"
+        src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-container-ts.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-xml.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-dash.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-hls.js"></script>
+    <script type="text/javascript"
+        src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-crypto.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-style.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-tizen.js"></script>
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-ui.js"></script>
+
+    <script type="text/javascript" src="js/console-to-ws.js"></script>
+</head>
+<button id="pipButtonElement">CLICK me to toggle PIP</button><div id="log" style="color: red;"></div>
+<br></br>
+<div id="my-player">
+    <video id="videoElement"></video>
+</div>
+
+<script>
+    // add all necessary (and loaded) modules to the player core
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.polyfill.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player['engine-bitmovin'].default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player['engine-native'].default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player['container-mp4'].default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player['container-ts'].default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.mserenderer.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.abr.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.drm.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.xml.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.dash.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.hls.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.crypto.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.style.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.tizen.default);

```

Comment:
Any reason why you add the Tizen module? I don't think this sample needs it, let's keep samples slim.